cmd/fyne/internal/commands/encode.go

```diff
@@ -0,0 +1,9 @@
+package commands
+
+import (
+	"html"
+)
+
+func encodeXMLString(in string) string {
+	return html.EscapeString(in)
```

Comment:
I don't think we need the wrapper function nor the tests if we are to use the built in function.

Reply:
Given that HTML and XML are not actually the same I felt the clarity was helpful.

Reply:
Good point